lib/sedna/common/log.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base logger"""
+
+import contextlib
+import functools
+import logging
+import time
+import threading
+import colorlog
+from sedna.common.config import BaseConfig
+
+
+loggers = {}
+
+
+class Logger(object):
+    """
+    Deafult logger in sedna
+    Args:
+        name(str) : Logger name, default is 'sedna'
+    """
+
+    def __init__(self, name: str = BaseConfig.job_name):
+        self.logger = logging.getLogger(name)
+
+        self.format = colorlog.ColoredFormatter(
+            '%(log_color)s[%(asctime)-15s] %(filename)s(%(lineno)d) [%(levelname)s]%(reset)s - %(message)s',
+        )
+
+        self.handler = logging.StreamHandler()
+        self.handler.setFormatter(self.format)
+
+        self.logger.addHandler(self.handler)
+        self.logLevel = 'INFO'
+        self.logger.setLevel(logging.INFO)
+        self.logger.propagate = False
+        self._is_enable = True
+
+    def disable(self):
+        self._is_enable = False
+
+    def enable(self):
+        self._is_enable = True
+
+    @property
+    def is_enable(self) -> bool:
+        return self._is_enable
+
+    def __call__(self, log_level, msg, *args, **kwargs):
+        if not self.is_enable:
+            return
+
+        self.logger.log(log_level, msg, *args, **kwargs)
+
+
+sednaLogger = Logger().logger
```

Comment:
sednaLogger rename into LOGGER or logger, without sedna word.

And here only use sednaLogger, then looks the other function of Logger is useless.

Reply:
fix